SparkPost/2web2ui
# FE-1172 & FE-1179 - [Domains UX Redesign] Domain Details Page

FE-1172 - [Domains UX Redesign] Domain Details Page 

### What Changed
 - Added a common "Domain Details" Page for Sending Domain, Bounce Domain and Tracking Domain.
 - Updated ExternalLink component to use a icon prop.
 - Added cypress tests for domain details page

### How To Test
 - point openresty to production, and use appteam in prod to test this.
 - _if(domain status === Blocked)_  mocks = https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/479035681/comment?frame-cb=1600868806904
_[the subtext for banner is a placeholder here]_
 -  _if( domain status === unverified or failed verification)_ mock =
https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/478910832/comment?frame-cb=1600868806904
 - _if( verified dkim && unverified bounce)_ mock =
https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/478910831?frame-cb=1600868806904
- _if( verified dkim && verified bounce && unverified spf)_ mock =
https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/479225549?frame-cb=1600868806904
- _if( domain is verified for sending  && verified for bounce && verified for spf )_ mock = 
https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/478945613?frame-cb=1600868806904
- _Check the mocks for tracking domains_ - https://sparkpost.invisionapp.com/console/share/CF19LDYPB2/478945611?frame-cb=1600868806904

- Check if all verification and authentication buttons work as expected.




### To Do
